qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -884,6 +897,12 @@ def program(self, program_id: str, refresh: bool = False) -> RuntimeProgram:
             RuntimeProgramNotFound: If the program does not exist.
             IBMRuntimeError: If the request failed.
         """
+        issue_deprecation_msg(
+            msg="Custom programs are being deprecated",
+            version="0.13.1",
```

Comment:
```suggestion
            version="0.14.0",
```

We need a minor version bump for a deprecation.